students/allen_maxwell/Lesson_05/database.py

```diff
@@ -0,0 +1,104 @@
+        '''Exits the connection'''
+        self.connection.close()
+
+def import_data(directory_name, product_file, customer_file, rentals_file):
+    '''
+    This function takes a directory name three csv files as input (product data,
+    customer data, and rentals data). It returns 2 tuples with a record count of the
+    number of data and errors for each file (product, customer, rentals).
+    '''
+    LOGGER.debug('Importing csv file')
+    mongo = MongoDBConnection()
+    with mongo:
+        database = mongo.connection.hp_norton
+
+         # Create collections
+        products = database[product_file]
+        customers = database[customer_file]
+        rentals = database[rentals_file]
+
+        products.drop
+        customers.drop
+        rentals.drop
+
+        prod_count, prod_errors = import_csv(directory_name, product_file, products)
+        cust_count, cust_errors = import_csv(directory_name, customer_file, customers)
+        rent_count, rent_errors = import_csv(directory_name, rentals_file, rentals)
+    return((prod_count, cust_count, rent_count), (prod_errors, cust_errors, rent_errors))
+
+def import_csv(directory_name, file_name, database):
+    '''
+    Creates and populates a new MongoDB database with data,
+    returns the number of files and errors
+    '''
+    LOGGER.debug('Importing %s', file_name)
+    errors = 0
+    count = 0
+    try:
+        file_csv = f'{file_name}.csv'
+        with open(os.path.join(directory_name, file_csv)) as file:
+            count = len(database.insert_many(csv.DictReader(file)).inserted_ids)
+    except (pyerror.InvalidOperation, FileNotFoundError) as error:
+        LOGGER.warning('Oops! something went wrong while reading %s: %s', file_csv, error)
+        errors += 1
+    return count, errors
+
+def show_available_products():
+    '''Returns a Python dictionary of products listed as available'''
+    results = {}
+    mongo = MongoDBConnection()
+    with mongo:
+        database = mongo.connection.hp_norton
+        LOGGER.info('Searching for available products')
+        for product in database.products.find({'quantity_available': {'$gt': '0'}}):
+            results[product['product_id']] = {
+                'description': product['description'],
+                'product_type': product['product_type'],
+                'quantity_available': product['quantity_available']}
+    return results
+
+def show_rentals(product_id):
+    '''
+    Returns a Python dictionary with the following user information from users that have
+    rented products matching the product id
+    '''
+    results = {}
+    mongo = MongoDBConnection()
+    with mongo:
+        database = mongo.connection.hp_norton
+        LOGGER.info('Searching for product_id: %s', product_id)
+        for rental in database.rentals.find({'product_id': product_id}):
+            for customer in database.customers.find({'user_id': rental['user_id']}):
+                results[customer['user_id']] = {

```

Comment:
Similar issue here, you could replace _database.rentals_ with _database['rentals']_ and _database.customers_ with _database['customers']_ then make it configurable so that _show_rentals_ instead uses _database['test_rentals']_ and _database['test_customers']_ for your tests.